continuous_integration/azure/install.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
It would have been nice to start with `git mv continuous_integration/travis/install.sh continuous_integration/azure/install.sh` before modifying the file, then the diff would be meaningful. From this file I can't tell what's been changed. Maybe it's not too late if you do something like:
```
git mv continuous_integration/travis/install.sh continuous_integration/azure/install.sh
git checkout continuous_integration/azure/install.sh
git reset
```
Maybe that would locally show the `git mv` operation plus a diff after the move?

Either way, the `continuous_integration/travis/*` files should go away

Reply:
Will do. I was worried I wouldn't be able to replicate the travis jobs perfectly because the astropy CI helper files were so detailed, so I was thinking I should keep both travis and azure for a bit to make sure the jobs were giving the same results. But I guess having the exact same miniconda version etc probably isn't that important...

Reply:
Not sure why the git rename tracking worked for the other 2 files but not for this one. I tried to get git to track it by renaming this file back to travis and then renaming to azure, with `git log` being:
```
commit 2488bfe0f9941dee4dfbcadafd4182772694c7f4 (HEAD -> azure, lucy/azure)
Author: Lucy Liu <jliu176@gmail.com>
Date:   Mon Jan 18 21:47:21 2021 +1100

    mv back to azure

 continuous_integration/{travis => azure}/install.sh | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
```
but it doesn't seem to register here....?

Reply:
I renamed the files again and added the changes in several commits. Github was showing the file as being renamed until the last commit. Would this be due to the proportion of lines changed..? Sorry but not sure what I can do about this.